.gitignore

```diff
@@ -27,3 +27,12 @@ config*.json
 # Pyinstaller files
 build
 dist
+
+# Environments
```

Comment:
It is very important to exclude venv-related files through .gitignore in Python projects.
However, each user may have a different folder name, and it is a different change from the big topic of this PR, so I recommend you to suggest it separately with a different PR.

Reply:
I completely forgot I added that to the .gitignore a few months ago. Definitely separate PR worthy, for sure

Reply:
Thank you for the update! However, in the process of removing the venv-related line, the existing new line is removed and marked as a change.
If you don't mind, could you add a newline to the .gitignore?